packages/react/src/__tests__/forwardRef-test.js

```diff
@@ -162,22 +162,32 @@ describe('forwardRef', () => {
     );
   });
 
-  it('should warn if the render function provided does not use the forwarded ref parameter', () => {
-    function arityOfZero() {
-      return null;
-    }
+  it('should not warn if the render function provided does not use any parameter', () => {
+    const arityOfZero = () => null;
```

Comment:
Can we make this more realistic? Such as a real pass through wrapper. Otherwise intent is not clear.